modules/apps/commerce/commerce-product-test/src/testFunctional/tests/SFCommerceOptionFacet.testcase

```diff
@@ -14,6 +14,42 @@ definition {
 		CommerceConfiguration.commerceTearDown();
 	}
 
+	@description = "This is a test for COMMERCE-10454."
```

Comment:
Can you add something more to the description that explains quickly what this test is doing?

Thanks!